src/components/OrderTable/OrderTable.tsx

```diff
@@ -36,10 +36,10 @@ function calculateOrderTotal(items: readonly CartItem[]) {
   return items.map(calculateItemTotal).reduce((sum, i) => sum + i, 0);
 }
 
-const OrderTable: React.FC<OrderTableProps> = ({ orderItems }) => {
+const OrderTable: React.FC<OrderTableProps> = () => {
   const { handleDecrementItem, handleRemoveItem, items } =
     useContext(CartContext);
-  const [cartItems, setCartItems] = useState<CartItem[]>(orderItems);
+  const [cartItems, setCartItems] = useState<CartItem[]>(items);
```

Comment:
Why do you need an extra `cartItems` variable?

Reply:
It is used for managing the local state in the component, and also for reflecting the current data in the cart context

Reply:
You don't need it :) use `items` from context

Reply:
fixed